tdrs-backend/tdpservice/users/models.py

```diff
@@ -43,6 +44,24 @@ def is_in_group(self, group_name: str) -> bool:
         """Return whether or not the user is a member of the specified Group."""
         return self.groups.filter(name=group_name).exists()
 
+    def save(self, *args, **kwargs):
+        """Prevent save if a attributes not nessisary for a user given their role."""
```

Comment:
Typos, recommended rewording:
"Prevent save if attributes are not necessary for a user given their role."